docs/source/modules/idealised_moist_phys.rst

```diff
@@ -0,0 +1,295 @@
+| Name                 | Description                                         | Units                              |
++======================+=====================================================+====================================+
+|``dt_ug_diffusion``   | Zonal wind tendency from vertical diffusion         | ms :math:`^{-2}`                   |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_vg_diffusion``   | Meridional wind tendency from vertical diffusion    | ms :math:`^{-2}`                   |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_tg_diffusion``   | Temperature tendency from vertical diffusion        | Ks :math:`^{-1}`                   |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_qg_diffusion``   | Specific humidity tendency from vertical diffusion  | kg kg :math:`^{-1}` s :math:`^{-1}`|
++----------------------+-----------------------------------------------------+------------------------------------+
+|``convection_rain``   | Rain from convection                                | kg m :math:`^{-2}` s :math:`^{-1}` |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``condensation_rain`` | Rain from large-scale condensation                  | kg m :math:`^{-2}` s :math:`^{-1}` |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``precipitation``     | Precipitation from resolved, parameterised and snow | kg m :math:`^{-2}` s :math:`^{-1}` |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_tg_convection``  | Temperature tendency from convection                | Ks :math:`^{-1}`                   |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_qg_convection``  | Specific humidity tendency from convection          | kg kg :math:`^{-1}` s :math:`^{-1}`|
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_tg_condensation``| Temperature tendency from convection                | Ks :math:`^{-1}`                   |
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_qg_condensation``| Specific humidity tendency from convection          | kg kg :math:`^{-1}` s :math:`^{-1}`|
++----------------------+-----------------------------------------------------+------------------------------------+
+|``dt_qg_condensation``| Specific humidity tendency from convection          | kg kg :math:`^{-1}` s :math:`^{-1}`|
```

Comment:
... tendency from condensation (not convection)

delete line 227 (same as 225)